inspirehep/modules/theme/views.py

```diff
@@ -101,3 +116,38 @@ def data():
 @blueprint.route('/ping')
 def ping():
     return 'OK'
+
+
+#
+# Feedback
+#
+
+@blueprint.route('/postfeedback', methods=['POST', ])
+def postfeedback():
+    """Handler to create a ticket from user feedback."""
+    feedback = request.form.get('feedback')
+    if feedback == '':
+        return jsonify(success=False), 400
+
+    replytoaddr = request.form.get('replytoaddr', None)
+    if replytoaddr is None:
+        if current_user.is_anonymous:
+            return jsonify(success=False), 403
+        else:
+            replytoaddr = current_user.get('email')
+            if replytoaddr == '':
+                return jsonify(success=False), 403
+
+    content = 'Feedback:\n{feedback}'.format(feedback=feedback)
+    message = {
+        'sender': current_app.config['CFG_SITE_SUPPORT_EMAIL'],
+        'recipients': [current_app.config['INSPIRELABS_FEEDBACK_EMAIL']],
+        'subject': 'INSPIRE Labs Feedback',
+        'body': content,
+        'reply_to': replytoaddr
+    }
+
+    if send_email.delay(message):
```

Comment:
Does the `delay` return value give an indication of success/failure? (http://docs.celeryproject.org/en/latest/reference/celery.app.task.html#celery.app.task.Task.delay)

Or it will always return a ` celery.result.AsyncResult` object that evaluates to `True` so the `if` will not have an effect.